packages/button/src/LionButton.js

```diff
@@ -240,8 +240,10 @@ export class LionButton extends DisabledWithTabIndexMixin(SlotMixin(LitElement))
    * of the form, and firing click on this button. This will fire the form submit
    * without side effects caused by the click bubbling back up to lion-button.
    * @param {Event} ev
-   * @private
+   * @protected
```

Comment:
__ so it's still private instead, right?

Reply:
We override this in our extension layer, and overriding private methods is not possible. So this method should actually be called '_clickDelegationHandler'

Since it's likely those methods are used by others atm, we delay the (breaking) name change until v1. Hence the comment `// TODO: rename to _clickDelegationHandler in v1`